### PR TITLE
POC - add hash router

### PR DESCRIPTION
**Pull request by amirfefer**

### CODE REVIEW

File: webpack/components/AnsibleHostDetail/AnsibleHostDetail.js [hunk 1]
```diff
@@ -1,32 +1,38 @@
...
-import { translate as __ } from 'foremanReact/common/I18n';
+import "./AnsibleHostDetail.scss";
 
-import './AnsibleHostDetail.scss';
-
-const AnsibleHostDetail = props => {
+const AnsibleHostDetail = (props) => {
   // https://projects.theforeman.org/issues/32398
-  const [activeTab] = useState('variables');
+  const [activeTab] = useState("variables");
```
Comment (xprazak2) [inline webpack/components/AnsibleHostDetail/AnsibleHostDetail.js:11]: No need to use state now for the active tab.

File: webpack/components/AnsibleHostDetail/AnsibleHostDetail.js [hunk 2]
```diff
@@ -1,32 +1,38 @@
...
-          <Label
-            color="blue"
-            icon={<InfoCircleIcon />}
-            style={{ marginTop: '1.5rem' }}
-          >
-            Ansible Variables coming soon!
-          </Label>
-        </Tab>
+          title={<TabTitleText>{__("Variables")}</TabTitleText>}
+          href="#Ansible/sub-tab"
```
Comment (xprazak2) [inline webpack/components/AnsibleHostDetail/AnsibleHostDetail.js:19]: Should this be `#Ansible/variables`?

File: webpack/components/AnsibleHostDetail/AnsibleHostDetail.js [hunk 3]
```diff
@@ -1,32 +1,38 @@
...
+        />
       </Tabs>
-    </div>
+      <Route path="/Ansible/sub-tab">
```
Comment (xprazak2) [inline webpack/components/AnsibleHostDetail/AnsibleHostDetail.js:22]: Same as above.

**Comment by xprazak2** [comment]
Good start! Needs a rebase.

File: webpack/components/AnsibleHostDetail/AnsibleHostDetail.js (hunk 2 above)
Comment (amirfefer) [reply webpack/components/AnsibleHostDetail/AnsibleHostDetail.js:19]: it's a poc :) 
if we will decide to use this approach I'll update this

**Pull request closed without merging**